draft/oss-support/post.md

```diff
@@ -0,0 +1,70 @@
+# OSS Support
```

Comment:
- Backbone Infrastructure, What's That???
  - What we take for granted.
  - I really should click that `donate` button sometime...
  - OpenSource Everywhere

I'll try harder in the morning.